### PR TITLE
CLIENT-4598 | Fix null incomingAllow bug and add test

### PR DESCRIPTION
**Pull request by ryan-rowland**
https://issues.corp.twilio.com/browse/CLIENT-4598

### CODE REVIEW

File: src/main/java/com/twilio/jwt/accesstoken/VoiceGrant.java [hunk 1]
```diff
@@ -82,9 +82,9 @@ public Object getPayload() {
          * @param  grant VoiceGrant
          */
         public Payload(VoiceGrant grant) {
-            if (grant.incomingAllow == true) {
+            if (grant.incomingAllow != null) {
               this.incoming = new HashMap<>();
-              this.incoming.put("allow", true);
+              this.incoming.put("allow", grant.incomingAllow);
```
Comment (cjcodes) [inline src/main/java/com/twilio/jwt/accesstoken/VoiceGrant.java]: Should we do a boolean check or cast here?

File: src/test/java/com/twilio/jwt/accesstoken/AccessTokenTest.java [hunk 2]
```diff
@@ -298,4 +298,37 @@ public void testVoiceToken() {
         Assert.assertEquals("AP123", outgoing.get("application_sid"));
         Assert.assertEquals("bar", outgoingParams.get("foo"));
     }
+
```
Comment (cjcodes) [inline src/test/java/com/twilio/jwt/accesstoken/AccessTokenTest.java]: Given that we changed the method above to pass through the value set, I think we should also test for values besides true or blank. Can we parameterize these tests and throw in a bunch of things like true, false, "", etc?

File: src/main/java/com/twilio/jwt/accesstoken/VoiceGrant.java (hunk 1 above)
Comment (ryan-rowland) [reply src/main/java/com/twilio/jwt/accesstoken/VoiceGrant.java]: `VoiceGrant.incomingAllow` is defined as a boolean, so I believe this should only be able to be true, false or null. Is that right?

File: src/test/java/com/twilio/jwt/accesstoken/AccessTokenTest.java (hunk 2 above)
Comment (ryan-rowland) [reply src/test/java/com/twilio/jwt/accesstoken/AccessTokenTest.java]: I don't think `""` should be possible, but I can add a test for false as well. I'm not familiar with how these tests are set up in Java, or how to parameterize them, but I'll look at some of the other tests and see if I can find an example.

File: src/main/java/com/twilio/jwt/accesstoken/VoiceGrant.java (hunk 1 above)
Comment (cjcodes) [reply src/main/java/com/twilio/jwt/accesstoken/VoiceGrant.java]: Oh, yup, you're right. I didn't dive deep enough into the chain. 👍

File: src/test/java/com/twilio/jwt/accesstoken/AccessTokenTest.java (hunk 2 above)
Comment (ryan-rowland) [reply src/test/java/com/twilio/jwt/accesstoken/AccessTokenTest.java]: @cjcodes How's this look?

**Pull request merged**